src/ServiceControl.Monitoring/Infrastructure/EndpointTracker.cs

```diff
@@ -1,15 +1,17 @@
 ﻿namespace ServiceControl.Monitoring.Infrastructure
 {
+    using System;
     using System.Threading.Tasks;
     using Messaging;
     using NServiceBus;
     using NServiceBus.Metrics;
 
     public class EndpointTracker : IHandleMessages<MetricReport>, IHandleMessages<LongValueOccurrences>, IHandleMessages<Occurrences>
     {
-        public EndpointTracker(EndpointRegistry endpointRegistry)
+        public EndpointTracker(EndpointRegistry endpointRegistry, EndpointInstanceActivityTracker activityTracker)
```

Comment:
Is the idea that any future metrics that get added will come through here as either a `LongValueOccurrences`, `MetricReport`, or `Occurrences`?

Or if it's a new type of metric report/occurrence that the handler will live in this class as well?